Library/Formula/pce.rb

```diff
@@ -7,8 +7,9 @@ class Pce < Formula
   head "git://git.hampa.ch/pce.git"
 
   devel do
-    url "http://www.hampa.ch/pub/pce/pre/pce-20140222-4b05f0c.tar.gz"
-    sha256 "44edaf071bb6840b6b3336174d528ff10c4dba8cb38194f0289fda81ac34f57f"
+    url "http://www.hampa.ch/pub/pce/pre/pce-20160308-72f1e10.tar.gz"
+    version "0.2.2+20160308"
```

Comment:
They seem to alternate version schemes between stable and unstable versions. For example, they had `20130218`  between `0.2.1` and `0.2.2`. I know prefixing version in this way may break some existing users for now, but the current devel version will prevent upgrading to stable versions later anyways. Another thing is that version scanning didn't work correctly with their filenames. The version was supposed to be `20140222`, but was set to `20140222-4` due to the following hash part.